src/babel.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+var _ = require('underscore-plus')
+var crypto = require('crypto')
+var path = require('path')
+var defaultOptions = require('../static/babelrc.json')
```

Comment:
It would be nice to inline this into Atom's root mega `package.json` file since all other `.json` and `.cson` files are currently placed there instead of bundled separately.